### PR TITLE
docs: added Funding and other news

### LINKED ISSUE #68: chore: update about us page
https://www.apiseven.com/zh/about

1. news: Financing
2. news: ingress controller GA

Welcome to discuss 

ping @moonming

### PR DESCRIPTION
**Pull request by juzhiyuan**
resolve #68

### CODE REVIEW

**Comment by juzhiyuan** [comment]
@membphis Ingress Controller is still under 1.0.0 milestone, is it in GA?

@moonming Please help to check the English translations.

File: data/about.json [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
+        "date": "2021 年 5 月",
+        "desc": "支流科技进入中国信通院发布的首批开源供应商名录"
+      },
+      {
+        "date": "2021 年 3 月",
+        "desc": "宣布获得顺为数百万美元 A 轮领投、真格基金跟投"
+      },
+      {
+        "date": "2021 年 2 月",
+        "desc": "宣布完成百万美元 Pre-A 轮融资"
```
Comment (moonming) [inline data/about.json:18]: 没有写机构名称

File: data/about.json [hunk 2]
```diff
@@ -59,9 +75,25 @@
...
+        "desc": "APISEVEN raises $1 Million in Series A Funding"
+      },
+      {
+        "date": "February 2021",
+        "desc": "APISEVEN raises $1 Million in Series Seed Funding"
+      },
       {
         "date": "December 2020",
-        "desc": "APISEVEN raises $1 Million in Series Angel Funding Led by ZhenFund"
+        "desc": "APISEVEN raises $1 Million in Pre-Seed Funding Led by ZhenFund"
```
Comment (moonming) [inline data/about.json:96]: 什么是 pre-seed？

**Review by moonming — CHANGES_REQUESTED** [review]

File: data/about.json (hunk 2 above)
Comment (juzhiyuan) [reply data/about.json]: https://support.crunchbase.com/hc/en-us/articles/115010458467-Glossary-of-Funding-Types#:~:text=Angel%3A%20An%20angel%20round%20is,groups%2C%20friends%2C%20and%20family.&text=Series%20A%20and%20Series%20B,between%20%241M%E2%80%93%2430M.

Just fixed the wrong type

File: data/about.json (hunk 1 above)
Comment (juzhiyuan) [reply data/about.json]: done

File: data/about.json [hunk 3]
```diff
@@ -59,9 +75,25 @@
...
+        "date": "May 2021",
+        "desc": "APISEVEN is one of the Open Source suppliers verified by CAICT"
+      },
+      {
+        "date": "March 2021",
+        "desc": "APISEVEN raises $1 Million in Series A Funding led by Shunwei Capital"
+      },
+      {
+        "date": "February 2021",
+        "desc": "APISEVEN raises $1 Million in Series Pre-A Funding led by Zhencheng Capital"
```
Comment (moonming) [inline data/about.json:92]: `$1 Million` -> 'millions'

File: data/about.json [hunk 4]
```diff
@@ -59,9 +75,25 @@
...
+        "date": "May 2021",
+        "desc": "APISEVEN open sourced the Service Mesh project based on Apache APISIX"
+      },
+      {
+        "date": "May 2021",
+        "desc": "APISEVEN is one of the Open Source suppliers verified by CAICT"
+      },
+      {
+        "date": "March 2021",
+        "desc": "APISEVEN raises $1 Million in Series A Funding led by Shunwei Capital"
```
Comment (moonming) [inline data/about.json:88]: ditto

File: data/about.json [hunk 5]
```diff
@@ -59,9 +75,25 @@
...
+      },
+      {
+        "date": "February 2021",
+        "desc": "APISEVEN raises $1 Million in Series Pre-A Funding led by Zhencheng Capital"
+      },
       {
         "date": "December 2020",
-        "desc": "APISEVEN raises $1 Million in Series Angel Funding Led by ZhenFund"
+        "desc": "APISEVEN raises $1 Million in Series Angel Funding led by ZhenFund"
```
Comment (moonming) [inline data/about.json:96]: ditto

File: data/about.json (hunk 3 above)
Comment (juzhiyuan) [reply data/about.json:92]: image

Just to make sure, do we need the extra `s`?

Comment (moonming) [reply data/about.json:92]: 融资金额是数百万美元，不是一百万美元

**Comment by juzhiyuan** [comment]
done

**Comment by juzhiyuan** [comment]
image

Just to make sure, those 2 dittos refer to the same case?

**Comment by juzhiyuan** [comment]
But I notice the news page says:

image

**Review by moonming — APPROVED** [review]

**Pull request merged**